### ISSUE TITLE
NMDP HLA Dictionary refresh

### ISSUE THREAD

**Issue by lgragert** [lgragert/nn-sero-pytorch#28]
This is different than refreshing the IMGT/HLA allele database.

More alleles will be added either to the training set or the validation set.

The training set are the "prototype alleles" - serology experts decide which alleles go here.
